frontend/libs/studio-components/src/components/StudioTableRemotePagination/utils.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { getRowsToRender } from './utils';
+
+describe('getRowsToRender', () => {
+  const rows = [
+    { id: 1, name: 'Row 1' },
+    { id: 2, name: 'Row 2' },
+    { id: 3, name: 'Row 3' },
+    { id: 4, name: 'Row 4' },
+    { id: 5, name: 'Row 5' },
+  ];
+
+  it('should return all rows when pageSize is 0', () => {
+    const currentPage = 1;
+    const pageSize = 0;
+    // eslint-disable-next-line testing-library/render-result-naming-convention
+    const rowsToRender = getRowsToRender(currentPage, pageSize, rows);
```

Comment:
Do we need to disable the eslint rule, because the rowsToRender includes the word `render`? 🤔

Reply:
Yes, it's weird. Might be possible to change the Eslint config to handle this.
https://github.com/testing-library/eslint-plugin-testing-library/issues/731

Reply:
[I have fixed this warning by adding a setting to .eslintrc.js. Please let me know if this is a better solution 😊 ](https://github.com/Altinn/altinn-studio/pull/12731/commits/0cdb8140a3ab307d2c0714453c2b76f1dd808df1)

Reply:
Maybe we should follow the rule and rename the variable to "rowsToDisplay"? 🤔